.cookiecutter/cookiecutter.json

```diff
@@ -4,7 +4,7 @@
   "directory": "pyapp",
   "ignore": [
       "bin/logger",
-      "conf/supervisord*.conf",
+      "conf/supervisord.conf",
```

Comment:
This removes only the live `supervisor.conf` which does nothing, and keeps the dev one which enables `make dev`.

This is required because the `tox` dev command assumes it's there.